docs/opensafely-cli.md

```diff
@@ -149,6 +149,18 @@ browser to access JupyterLab.
 To exit, press ++control+c++ in the command line - this also shuts down the container.
 Or alternatively go to File -> shutdown in the JupyterLab tab.
 
+### `unzip` - unzipping csv files
+
+For performance and storage reasons on the backend, you need to use the the
```

Comment:
```suggestion
For performance and storage reasons on the backend, you must use the
```
nit - preference for requirement levels from https://www.rfc-editor.org/rfc/rfc2119

Reply:
(also type double `the`)